docs/UserGuide.md

```diff
@@ -243,3 +272,4 @@ _Details coming soon ..._
 | **Today**  | `today`                                                                                                                                                                                                                          |
 | **Week**   | `week {WEEK_NUMBER \| first \| next \| prev \| last}` <br> e.g., `week 2`                                                                                                                                                        |
 | **Year**   | `year {YEAR_NUMBER \| next \| prev}` <br> e.g., `year 2022`                                                                                                                                                                      |
+>>>>>>> 659fd5e5c4f9b6891484018aaeef779e57f59497
```

Comment:
Here's the end of the merge conflict marker, you're supposed to tell them which one you want to marge in because Github couldn't do this automatically